components/brave_today/common/features.cc

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_today/common/features.h"
+
+#include "base/feature_list.h"
+#include "brave/components/brave_today/buildflags/buildflags.h"
+
+namespace brave_today {
+namespace features {
+
+const base::Feature kBraveNewsFeature{
+  "BraveNews",
+
+#if BUILDFLAG(ENABLE_BRAVE_NEWS) && defined(OS_ANDROID)
+      base::FEATURE_ENABLED_BY_DEFAULT
+#else
+      base::FEATURE_DISABLED_BY_DEFAULT
+#endif
```

Comment:
Is this intentional to enable by default on Androiid?

Reply:
yes, i thought so since we will be keeping it in Nightly for now. @petemill does this look ok to you?

Reply:
perhaps we should disable them by default and enable in nightly channel via griffin only?

Reply:
yes, how about this:
Enable by default for non-android (i.e. desktop)
Enable by default for local / non-official build (for android local development)
Disable by default for android

we will then enable on android nightly via griffin

Please manually test with flag on and off to check NTP doesn't crash.

Reply:
done